src/Microsoft.DocAsCode.Build.Engine/DocumentBuildContext.cs

```diff
@@ -193,6 +201,26 @@ private List<string> ResolveByExternalReferencePackages(List<string> uidList, Co
             return list;
         }
 
+        private XRefSpec FindByRequest(string uid)
+        {   
+            
+            XRefSpec xf = null;
+            string requestUrl = XRefServerUrls[0];
```

Comment:
what if it contains more than 1 value?

---
In reply to: [126058914](https://github.com/dotnet/docfx/pull/1837#discussion_r126058914) [](ancestors = 126058914)